pyccel/errors/messages.py

```diff
@@ -73,7 +73,7 @@
 PYCCEL_RESTRICTION_TRY_EXCEPT_FINALLY = 'Uncovered try/except/finally statements by Pyccel'
 PYCCEL_RESTRICTION_RAISE = 'Uncovered raise statement by Pyccel'
 PYCCEL_RESTRICTION_YIELD = 'Uncovered yield statement by Pyccel'
-PYCCEL_RESTRICTION_IS_RHS = 'Only booleans and None are allowed as rhs for is statement'
+PYCCEL_RESTRICTION_IS_RHS = 'Only booleans, integers and None are allowed as rhs for is statement'
```

Comment:
Maybe you should improve this error message, since we made `is` symmetrical there is no difference between rhs and lhs